README.md

```diff
@@ -55,6 +55,8 @@ generic with minimal need for project-specific behavior.
 
 - AWS credentials configured in `~/.aws/credentials` and/or `~/.aws/config`
 
+- git-secrets
```

Comment:
```suggestion
- git-secrets
```

This should link to the section added below. Or inline that section here directly in the bullet point.